definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -196,9 +196,11 @@ declare module 'express' {
     use(middleware: Middleware): this;
     use(...middelware: Array<Middleware>): this;
     use(path: string, ...middelware: Array<Middleware>): this;
+    use(path: string, router: Router): this;
 
     route(path: string): Route;
   }
+  declare type $Router = Router;
```

Comment:
Ah, I misunderstood it. I thought that I needed to define a special type `$Router` to use `import type { $Router }`, but it was unnecessary. Thank you! I'll remove it.